libs/etterlatte-funksjonsbrytere/src/main/kotlin/no/nav/etterlatte/funksjonsbrytere/FeatureToggleService.kt

```diff
@@ -16,29 +16,36 @@ interface FeatureToggleService {
     ): Boolean
 
     companion object {
-        fun initialiser(properties: FeatureToggleProperties) = UnleashFeatureToggleService(properties)
+        fun initialiser(
+            properties: FeatureToggleProperties,
+            brukerIdent: () -> String = { "" },
+        ) = UnleashFeatureToggleService(properties, brukerIdent)
     }
 }
 
-class UnleashFeatureToggleService(private val properties: FeatureToggleProperties) : FeatureToggleService {
+class UnleashFeatureToggleService(
+    private val properties: FeatureToggleProperties,
+    private val brukerIdent: () -> String,
+) : FeatureToggleService {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
     private val defaultUnleash =
         DefaultUnleash(
             UnleashConfig.builder()
                 .appName(properties.applicationName)
                 .unleashAPI(properties.uri())
-                .unleashContextProvider(lagUnleashContextProvider())
+                .unleashContextProvider(lagUnleashContextProvider(brukerIdent))
                 .apiKey(properties.apiKey)
                 .build(),
             GradualRolloutRandomStrategy(),
             GradualRolloutUserIdStrategy(),
         )
 
-    private fun lagUnleashContextProvider() =
+    private fun lagUnleashContextProvider(brukerIdentResolver: () -> String) =
         UnleashContextProvider {
             UnleashContext.builder()
                 .appName(properties.applicationName)
+                .userId(brukerIdentResolver())
```

Comment:
Trur userid med denne endringa går frå å vera null til å vera "" der det ikkje er overstyrt, men aner ikkje om det har nokon praktisk konsekvens.

Reply:
gjorde om bittelitt for å benytte `null` som default